src/AspNet.Security.OpenIdConnect.Server/Events/AuthorizationEndpointContext.cs

```diff
@@ -13,15 +13,15 @@ namespace AspNet.Security.OpenIdConnect.Server {
     /// An event raised after the Authorization Server has processed the request, but before it is passed on to the web application.
     /// Calling RequestCompleted will prevent the request from passing on to the web application.
     /// </summary>
-    public sealed class AuthorizationEndpointNotification : BaseNotification<OpenIdConnectServerOptions> {
+    public sealed class AuthorizationEndpointContext : BaseControlContext<OpenIdConnectServerOptions> {
         /// <summary>
         /// Creates an instance of this context
         /// </summary>
-        internal AuthorizationEndpointNotification(
-            HttpContext context,
+        internal AuthorizationEndpointContext(
+            HttpContext httpContext,
```

Comment:
I'm not sure to understand why this change is needed. Please revert all these changes :+1: